src/app/core/models/label.model.ts

```diff
@@ -1,4 +1,25 @@
-export interface Label {
-    labelValue: string;
-    labelColor: string;
+export class Label {
+
+  labelValue: string;
+  labelColor: string;
+  labelCategory: string;
```

Comment:
What's the reason for this ordering? Follow a consistent order. In the constructor, the parameters are ordered differently.